snmp-mixin/alerts/alerts.libsonnet

```
@@ -28,6 +28,18 @@
             },
             'for': '5m',
           },
+          {
+            alert: 'SNMPRestarted',
+            expr: 'sysUpTime < 60',
+            labels: {
+              severity: 'warning',
+            },
+            annotations: {
+              summary: 'SNMP Target Restarted',
```

```suggestion
              summary: 'SNMP target restarted',
```
according to our style guide.

---

Corrected